packages/angular/package.json

```diff
@@ -58,7 +56,11 @@
     "webpack-merge": "5.7.3"
   },
   "peerDependencies": {
-    "@nguniversal/builders": "~15.1.0",
+    "@angular-devkit/architect": ">= 0.1400.0 < 0.1600.0",
+    "@angular-devkit/schematics": ">= 14.0.0 < 16",
+    "@schematics/angular": ">= 14.0.0 < 16",
+    "@angular-devkit/core": ">= 14.0.0 < 16",
+    "@nguniversal/builders": ">= 14.0.0 < 16",
```

Comment:
I'd say we should add `@angular-devkit/build-angular` as well for correctness. We depend on it directly and we already install it when initiating the plugin.